### DISCUSSION TITLE
Supabase Auth

### DISCUSSION THREAD

**Discussion by blaszta** [lowcoder-org/lowcoder#1015] [Ideas]
Any plan to integrate Supabase Auth in the future?

**Comment by FalkWolsky** [comment]
Yes. As Supabase can use OpenID providers - but does not act by itself as such - it is a bit more complex, as we need to implement it I a way, that we accept Supabase as Auth Authority - without standard mechanism of OAuth or OpenID. But we would like to create this bridge, as we also see much value in this combination.

**Comment by FalkWolsky** [comment]
We did integrate the Supabase SDK since version 2.4. Now you can make use of it directly in Lowcoder.

<img width="518" alt="Screenshot 2025-01-20 at 00 28 06" src="https://github.com/user-attachments/assets/836d18b0-6f00-4d38-af62-248f122e5db3" />

**Comment by blaszta** [comment]
That's awesome! Thanks!
